tests/golden/billing/appcat/appcat/billing/10_odoo_secret.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+data: {}
+kind: Secret
+metadata:
+  annotations: {}
+  labels:
+    app.kubernetes.io/managed-by: commodore
+    app.kubernetes.io/name: appuio-reporting
+    app.kubernetes.io/part-of: syn
+    name: odoo-credentials
+  name: odoo-credentials
+  namespace: syn-appcat
+stringData:
+  client_id: t-silent-test-1234/c-green-test-1234/billing-collector-cloudservices/odoo-oauth-client-id
+  client_secret: t-silent-test-1234/c-green-test-1234/billing-collector-cloudservices/odoo-oauth-client-secret
+  token_endpoint: https://test.central.vshn.ch/api/v2/authentication/oauth2/token
```

Comment:
please double check those values as well

Reply:
This is golden test links, they are not to be deployed. Moreover the setup has been configured for APPUiO Cloud which is working

Reply:
yeah I understand, but we add those IDs and Secrets per cluster, this is why I thought it's worth checking in vault if they're all present on each cluster under the same path

Reply:
but we can ignore that, monitoring would catch issues with exporter

Reply:
> yeah I understand, but we add those IDs and Secrets per cluster

no, that's wrong. Unfortunately there's a bit of confusion with our billing setup. The only thing we deploy on each APPUiO Managed cluster is a Prometheus Rule which scrapes specific metrics and sends to Mimir. This operation does not require any authorization/authentication. The real billing setup is only deployed on APPUiO cloud lpg2. There the setup already exists and is configured, I just add more cronjobs to the bunc of cronjobs that already exist. The auth is already working otherwise our APPUiO cloud services would not get billed.

Reply:
ah yes, that makes sense. Resolved.